src/features/Missions/Missions.css

```diff
@@ -0,0 +1,71 @@
+:root {
+  --border: 1px solid #5e6062;
+}
+
+.missions {
+  padding: 25px;
+  display: flex;
+  justify-content: center;
+}
+
+.mission-name {
+  font-weight: bold;
+}
+
+table {
+  width: 90%;
+  border-right: var(--border);
+  border-top: var(--border);
+}
+
+tr:nth-child(odd) {
+  background-color: #f1f2f2;
+}
+
+th {
+  background-color: white !important;
+  text-align: center;
+  padding: 20px;
+  border-left: var(--border);
+  border-bottom: var(--border);
+}
+
+td {
+  padding: 20px;
+  border-left: var(--border);
+  border-bottom: var(--border);
+}
+
+.mstatus {
+  width: max-content;
+}
+
+td button {
+  width: max-content;
+  padding: 8px;
+  border-radius: 5px;
+}
+
+.mstat {
+  border-radius: 5px;
+  padding: 5px;
+  width: max-content;
+  color: white;
+  font-weight: bold;
+}
+
+.member {
+  background-color: rgb(33, 168, 185);
+}
+
+.notmember {
+  background-color: rgb(70, 72, 72);
+}
+
+.leavebtn {
+  border: 2px solid red;
+}
```

Comment:
- [ ] Good job so far but as you can notice the `Leave Mission` button the text is in red as well, kindly fix that 🙏🏾 

Your design

<img width="1036" alt="Screenshot 2023-05-18 at 16 32 00" src="https://github.com/Amazinggracee/rocket-mission/assets/74501523/69244578-9ecc-47a5-8c92-63eb47d211a1">

Original design

<img width="698" alt="Screenshot 2023-05-18 at 16 38 22" src="https://github.com/Amazinggracee/rocket-mission/assets/74501523/29150f79-cc52-443a-a36d-72b9e8ee7ff7">